src/windows/component.rs

```diff
@@ -157,11 +158,12 @@ impl Connection {
     fn new() -> Option<Connection> {
         // "Funnily", this function returns ok, false or "this function has already been called".
         // So whatever, let's just ignore whatever it might return then!
-        unsafe { CoInitializeEx(null_mut(), 0) };
+        let val = unsafe { CoInitializeEx(null_mut(), 0) };
         Some(Connection {
             instance: None,
             server_connection: None,
             enumerator: None,
+            inited: if val == 0 || val == 1 { true } else { false },
```

Comment:
Please use the constant instead of `0` and `1`. Also, no need for the if else condition, just having `val == 0 || val == 1` is enough.

Reply:
done